test/modules/TestResultCoordinator/Reports/DeploymentTestResultComparer.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// This test result comparer is used to determine if 2 deployment test result matches.
```

Comment:
Add to the comment that matches means that all values from expected are included in actual, but not required the other way around

Reply:
good point.